### ISSUE TITLE
win_iis.deployed does not update values

### ISSUE THREAD

**Issue by iHarshalAgarwal** [saltstack/salt#59780]
**Description**
win_iis.deployed does not update the values, if the site exists in IIS with some other configuration settings.

**Setup**
**pillar:**
  - siteName: 'mysite'
    sourcePath: 'C:\\inetpub\\mysite
    appPool: 'somepool'
    port: 81
    protocol: 'http'

**salt:**
add_{{ sites['siteName'] }}_site":
  win_iis.deployed:
    - name: {{ sites['siteName'] }}
    - sourcepath: {{ sites['sourcePath'] }}
    - apppool: {{ sites['appPool'] }}
    - port: {{ sites['port'] }}
    - protocol: {{ sites['protocol'] }}

**Steps to Reproduce the behavior**
1. Run the above state with some values.
2. Expect the site to be created.
3. Update port from 81 to 82 and rerun

**Expected behavior**
It should update the port for that site in IIS

**Screenshots**
Result: comment: Site already present: mysite

**Versions Report**
master version: 2017.7.4 (Nitrogen) - this is on Ubuntu
minions version: - 3002.5 - this is on windows

**Comment by sagetherage** [comment]
@iHarshalAgarwal your master is running an unsupported version of Salt are you able to upgrade to something that is supported? >v3000 See our [Platform Support Page](https://saltproject.io/salt-platform-support/) for details

**Comment by iHarshalAgarwal** [comment]
I'll look at it. Thanks @sagetherage

**Comment by twangboy** [comment]
Yeah, looks like the state only checks for the presence of the site and nothing else.
